projects/pswg/src/main/java/com/parzivail/pswg/entity/ThermalDetonatorEntity.java

```diff
@@ -122,9 +107,9 @@ protected void onCollision(HitResult hitResult)
 	@Override
 	public ActionResult interact(PlayerEntity player, Hand hand)
 	{
-		if (!isPrimed() && age > 30)
+		if (!isPrimed() && age > 30 && player.getInventory().getMainHandStack().isEmpty())
```

Comment:
replace `30` magic number with a named constant like before that describes why the value is 30, like `MIN_PICKUP_AGE` if that's what this is for